Description: Hardcode actions
 From debian/changelog for version 1.1-1:
 .
 Modified to work with Debian PPP scripts and Debian ip-{up,dow}.d
 structure.
 .
 Converted to quilt for version 1.3.0+20120605-1.
Author: Marcelo E. Magallon <mmagallo@debian.org>
Last-Update: 2014-09-12

--- a/wmppp/wmppp.c
+++ b/wmppp/wmppp.c
@@ -187,10 +187,10 @@
  /* Defines */
 /***********/
 
-#define START_ACTION (NULL)
-#define STOP_ACTION (NULL)
+#define START_ACTION "/usr/bin/pon"
+#define STOP_ACTION "/usr/bin/poff"
 #define SPEED_ACTION (NULL)
-#define IFDOWN_ACTION (NULL)
+#define IFDOWN_ACTION "/usr/bin/poff -r"
 
 #define STAMP_FILE "/var/run/ppp0.pid"
 
